src/T2DMap.cpp

```diff
@@ -2645,61 +2644,61 @@ void T2DMap::mousePressEvent(QMouseEvent* event)
 
-            auto recolorRoom = new QAction("color", this);
+            auto recolorRoom = new QAction(tr("Color", "Menu option to change a room color in the mapper"), this);
             recolorRoom->setToolTip(tr("Change room color"));
             connect(recolorRoom, SIGNAL(triggered()), this, SLOT(slot_changeColor()));
 
-            auto spreadRooms = new QAction("spread", this);
+            auto spreadRooms = new QAction(tr("Spread", "Menu option to space out rooms in the mapper"), this);
             spreadRooms->setToolTip(tr("Increase map X-Y spacing for the selected group of rooms"));
             connect(spreadRooms, SIGNAL(triggered()), this, SLOT(slot_spread()));
 
-            auto shrinkRooms = new QAction("shrink", this);
+            auto shrinkRooms = new QAction(tr("Shrink", "Menu option to move rooms closer in the mapper"), this);
             shrinkRooms->setToolTip(tr("Decrease map X-Y spacing for the selected group of rooms"));
             connect(shrinkRooms, SIGNAL(triggered()), this, SLOT(slot_shrink()));
 
-            auto lockRoom = new QAction("lock", this);
+            auto lockRoom = new QAction(tr("Lock", "Menu option to lock a room for speed walks in the mapper"), this);
             lockRoom->setToolTip(tr("Lock room for speed walks"));
             connect(lockRoom, SIGNAL(triggered()), this, SLOT(slot_lockRoom()));
 
-            auto unlockRoom = new QAction("unlock", this);
+            auto unlockRoom = new QAction(tr("Unlock", "Menu option to unlock a room for speed walks in the mapper"), this);
             unlockRoom->setToolTip(tr("Unlock room for speed walks"));
             connect(unlockRoom, SIGNAL(triggered()), this, SLOT(slot_unlockRoom()));
 
-            auto weightRoom = new QAction("weight", this);
+            auto weightRoom = new QAction(tr("Weight", "Menu option to set a room wight in the mapper"), this);
             weightRoom->setToolTip(tr("Set room weight"));
             connect(weightRoom, SIGNAL(triggered()), this, SLOT(slot_setRoomWeight()));
 
-            auto roomExits = new QAction("exits", this);
+            auto roomExits = new QAction(tr("Exits", "Menu option to set exits for a room in the mapper"), this);
             roomExits->setToolTip(tr("Set room exits"));
             connect(roomExits, SIGNAL(triggered()), this, SLOT(slot_setExits()));
 
-            auto roomSymbol = new QAction("symbol", this);
+            auto roomSymbol = new QAction(tr("Symbol", "Menu option to set symbol(s) to mark rooms in the mapper"), this);
             roomSymbol->setToolTip(tr("Set one or more symbols or letters to mark special rooms"));
             connect(roomSymbol, SIGNAL(triggered()), this, SLOT(slot_setSymbol()));
 
-            auto moveRoomXY = new QAction("move to", this);
+            auto moveRoomXY = new QAction(tr("Move to", "Menu option to move rooms elsewhere in the mapper"), this);
```

Comment:
`"Menu option to move rooms elsewhere in the`**`current area`**`in the mapper"`

Reply:
Do you really think a translation may be different, if a translator mistakenly assumes, a room can also be moved to a different area by clicking "move to"?

Reply:
Well there is a different option for **that** (moving the rooms to a different area) and I'd have hoped the disambiguation comment would help to, erm, *disambiguate* things... :stuck_out_tongue_winking_eye:

Reply:
I think it does though.

Reply:
Okay - lets suck it and see.